spec/spec_helper.rb

```diff
@@ -76,3 +76,10 @@
 require File.dirname(__FILE__) + "/helpers/factories.rb"
 
 ActiveRecord::Base.logger = Logger.new(STDOUT) if ENV["SQL"]
+
+DBQueryMatchers.configure do |config|
```

Comment:
I've added some ignores to master branch too 🙂

Reply:
Sorry, I'll remove it. Thanks for pointing.

Reply:
no worries, we can do it during merge